core/src/main/java/org/jivesoftware/sparkimpl/certificates/CertificateController.java

```diff
@@ -128,29 +134,15 @@ public void addOrRemoveFromExceptionList(boolean checked){
 	
 	/**
 	 * Return information if certificate is on exception list.
+	 * 
 	 * @param alias
-	 * @return
 	 */
-	public boolean isOnExceptionList(String alias){
-		try (FileInputStream input = new FileInputStream(EXCEPTIONS)) {
-			trustStore = KeyStore.getInstance("JKS");
-			trustStore.load(input, passwd);
-
-			Enumeration store = trustStore.aliases();
-			while (store.hasMoreElements()) {
-				String searchAlias = (String) store.nextElement();
-				X509Certificate certificate = (X509Certificate) trustStore.getCertificate(alias);
-				if (alias.equals(searchAlias)) {
-					return true;
-				}
-			}
-			} catch (IOException | KeyStoreException | NoSuchAlgorithmException | CertificateException e) {
-				Log.error("Cannot access Keystore", e);
-			}
-		
-			return false;
-	}
-	
+	public boolean isOnExceptionList(CertificateModel cert) {
```

Comment:
You can simplify this a bit. :)

``return exemptedCertificates.contains(cert);``